apps/src/javalab/control-buttons.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'color.scss';
+
+@mixin controlButton {
+  float: left;
+  font-size: 15px;
+  width: 140px;
+  background-color: $orange;
+  border-color: $orange;
+  font-family: "Gotham 5r";
+  padding: 5px 12px;
+  margin: 5px 0 5px 5px;
+  justify-content: center;
+
+  &:hover {
+    color: $white;
+    box-shadow: none;
+  }
+}
+
+@mixin whiteControlButton {
+  @include controlButton;
+  background-color: $white;
+  border-color: $dark_charcoal;
+  color: $dark_charcoal;
+
+  &:hover {
+    color: $dark_charcoal;
+  }
+}
+
+.button {
+  &Orange {
```

Comment:
nit: I've typically always seen class names lowercase and prefixed with `.`, i.e. `&.orange`, unless we have a specific stylistic preference otherwise?

Reply:
SCSS turns this into a class called .buttonOrange -- a little more discussion on that convention here:

https://github.com/code-dot-org/code-dot-org/pull/47508#discussion_r940461006

Reply:
Oh neat! Didn't know about this, thanks 👍